react-native-libraries.json

```diff
@@ -8864,5 +8864,20 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Stringsaeed/react-native-tooltiplize",
+    "npmPkg": "https://www.npmjs.com/package/react-native-tooltiplize",
```

Comment:
```suggestion
```

This field only should contain npm package name to validate correctly, but it is not needed when GitHub repository name matches package name.

Reply:
Thanks for the update!